### PR TITLE
add script that installs s4tf and deps on ubuntu1804

### PR DESCRIPTION
**Pull request by marcrasi**
The file comment describes what the script does and what the options are.

The purposes of this script are:
* Make it easy to automatedly install S4TF on Ubuntu 18.04 so that automated S4TF jobs can be written.
* Factor this logic out of the existing Dockerfiles that we have in various repos. (After this lands, I'll gradually refactor the Dockerfiles to refer to this).
* We could refer to this in installation instructions.

### CODE REVIEW

File: utils/install-ubuntu1804.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+#                    nightly CPU-only toolchain.
+#   --jupyter-url: Specifies the URL for swift-jupyter. Defaults to the latest
+#                  nightly build. Set this to the empty string to disable
+#                  swift-jupyter installation.
+#   --install-location: Directory to extract the toolchain. Defaults to
+#                       "./swift-toolchain".
+
+set -exuo pipefail
+
+TOOLCHAIN_URL=https://storage.googleapis.com/swift-tensorflow-artifacts/nightlies/latest/swift-tensorflow-DEVELOPMENT-ubuntu18.04.tar.gz
```
Comment (yashk2810) [inline utils/install-ubuntu1804.sh]: Is this available for ubuntu 16.04 too?

Comment (marcrasi) [reply utils/install-ubuntu1804.sh]: No, our automated builds only do 18.04. The full list of urls for all the versions we publish is here: https://github.com/tensorflow/swift/blob/master/Installation.md

**Comment by dynamicwebpaige** [comment]
Pinging @dan-zheng for a review, and @saeta for visibility. Thanks for creating this, @marcrasi! 😄

**Comment by yashk2810** [comment]
We now have Ubuntu 18.04 machines. Can we get this submitted so I can play around with it? :)

**Comment by dan-zheng** [comment]
CI failed:
```
...
Step 8/12 : ENV PATH="/swift-tensorflow-toolchain/usr/bin:${PATH}"
 ---> Running in a21be64c8e10
Removing intermediate container a21be64c8e10
 ---> c34073a35c69
Step 9/12 : WORKDIR /swift
 ---> Running in 99c393143efb
Removing intermediate container 99c393143efb
 ---> 1a420755994f
Step 10/12 : COPY . .
 ---> a5dbd768d6b5
Step 11/12 : RUN swift build -Xswiftc -enable-experimental-quasiquotes
 ---> Running in 4954f273b6c3
error: root manifest not found
The command '/bin/sh -c swift build -Xswiftc -enable-experimental-quasiquotes' returned a non-zero code: 1
```

@marcrasi: wonder if you know what's going on? `error: root manifest not found` suggests that there's no `Package.swift` in the working directory.

**Comment by marcrasi** [comment]
Oh it's probably just that this branch is so old that CI is confused. I'll update the branch and see if CI works.

Also, since it's been a really long time since I made this, the script may be out of date in some ways. I'll retest it and upload fixed before merging.

**Pull request merged**